baseapp/test_helpers.go

```diff
@@ -17,13 +17,19 @@ func (app *BaseApp) Check(txEncoder sdk.TxEncoder, tx sdk.Tx) (sdk.GasInfo, *sdk
 	}
 	ctx := app.checkState.ctx.WithTxBytes(bz).WithVoteInfos(app.voteInfos).WithConsensusParams(app.GetConsensusParams(app.checkState.ctx))
 	gasInfo, result, _, _, err := app.runTx(ctx, runTxModeCheck, bz)
+	if len(ctx.MultiStore().GetEvents()) > 0 {
+		panic("Expected checkTx events to be empty")
```

Comment:
does it make sense to panic if its just events? this isn't on the critical path so it seems strange to error out b/c of this. Perhaps a warning log instead?

Reply:
yeah, this is just unit test

Reply:
oo ops